### PR TITLE
Hyperledger AnonCreds 2023Q2 Quarterly Report

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+[AnonCreds Specification] adjusted to align with the implementation. Happily, the
+changes were relatively small and helped to clarify the specification.
+
+## Questions/Issues for the TSC
+
+None.
+
+## Releases
+
+- [anoncreds-rs] -- [v0.1.0-dev.1] through [v0.1.0-dev12] (12 releases)
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md]:
```suggestion
- [anoncreds-rs] -- [v0.1.0-dev.1] through [v0.1.0-dev.12] (12 releases)
```

File: project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+  and using [did:web](https://learn.mattr.global/tutorials/dids/did-web).
+- As adjustments to the implementation were made, alignment was made with the
+  AnonCreds v1.0 specification. The adjustments were minor and were done such that
+  existing, issued AnonCreds credentials would still be useable.
+- Work began in the AnonCreds v2.0 specification (see below for details)
+
+An AnonCreds v2.0 Working Group has begun meeting bi-weekly to discuss the "next
+generation" of AnonCreds.  So far we have discussed the AnonCreds data objects,
+including what would be published by the issuer to the Ledger, credentials, and
+presentations, plus a potential revocation scheme ([ALLOSSAUR], [ALLOSAUR Summary]) that could be
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md]:
```suggestion
presentations, plus a potential revocation scheme ([ALLOSAUR], [ALLOSAUR Summary]) that could be
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]

**Review by TimoGlastra — APPROVED** [review]

**Review by arsulegai — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+## Releases
+
+- [anoncreds-rs] -- [v0.1.0-dev.1] through [v0.1.0-dev.12] (12 releases)
+
+[v0.1.0-dev.1]: https://github.com/hyperledger/anoncreds-rs/releases/tag/v0.1.0-dev.1
+[v0.1.0-dev.12]: https://github.com/hyperledger/anoncreds-rs/releases/tag/v0.1.0-dev.12
+
+## Overall Activity in the Past Quarter
+
+The AnonCreds project has has significant progress on three fronts this quarter:
```
Comment (VRamakrishna) [inline project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md]: "has has" --> "has made"

File: project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+[Aries Framework JavaScript]: https://github.com/hyperledger/aries-framework-javascript
+[Aries Cloud Agent Python]: https://github.com/hyperledger/aries-cloudagent-python
+
+## Current Plans
+
+Work continues on getting a final `v0.1.0` release completed with the details in
+the wrappers stabilized. With the end-of-life of the Hyperledger Ursa project, a
+new AnonCreds repository is needed to host the CL-Signatures code used in
+[anoncreds-rs], and producing artifacts from that new repository. In theory,
+that should be a straight forward transfer of code from the existing Ursa
```
Comment (VRamakrishna) [inline project-reports/2023/2023-Q2-Hyperledger-AnonCreds.md]: "straight forward" --> "straightforward".

**Review by tkuhrt — APPROVED** [review]

**Review by VRamakrishna — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**